lib/outbrain/api/report.rb

```diff
@@ -0,0 +1,11 @@
+module Outbrain
+  module Api
+    class Report < Base
+      PATH = 'performanceByCampaign'
```

Comment:
seeing as this path is specific to a campaign/marketer might now want this in the base report class